app/pages/leadtrustee/individual/NinoYesNoPage.scala

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package pages.leadtrustee.individual
+
+import models.UserAnswers
+import pages.QuestionPage
+import play.api.libs.json.JsPath
+
+import scala.util.Try
+
+case object NinoYesNoPage extends QuestionPage[Boolean] {
```

Comment:
Despite it being poorly named, I believe the already existing `UkCitizenPage` serves the same purpose as this.